### PR TITLE
Case 21199: Fix borken shaders for forward rendering (quest)

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.manuscript.com/f/cases/21199/Fix-broken-shaders-used-by-key-ui-element-laser-teleport

### CODE REVIEW

File: libraries/entities-renderer/src/RenderablePolyLineEntityItem.cpp [hunk 1]
```diff
@@ -29,6 +29,13 @@ gpu::PipelinePointer PolyLineEntityRenderer::_glowPipeline = nullptr;
 
 static const QUrl DEFAULT_POLYLINE_TEXTURE = PathUtils::resourcesUrl("images/paintStroke.png");
 
+#if defined(USE_GLES)
+static bool DISABLE_DEFERRED = true;
+#else
+static const QString RENDER_FORWARD{ "HIFI_RENDER_FORWARD" };
+static bool DISABLE_DEFERRED = QProcessEnvironment::systemEnvironment().contains(RENDER_FORWARD);
+#endif
```
Comment (SamGondelman) [inline libraries/entities-renderer/src/RenderablePolyLineEntityItem.cpp]: Maybe we could move this to its own file and just include it in every .cpp where we need it?

File: libraries/render-utils/src/RenderForwardTask.cpp [hunk 2]
```diff
@@ -101,7 +101,6 @@ void RenderForwardTask::build(JobModel& task, const render::Varying& input, rend
     const auto inFrontOpaquesInputs = DrawLayered3D::Inputs(inFrontOpaque, lightingModel, nullJitter).asVarying();
     const auto inFrontTransparentsInputs = DrawLayered3D::Inputs(inFrontTransparent, lightingModel, nullJitter).asVarying();
     task.addJob<DrawLayered3D>("DrawInFrontOpaque", inFrontOpaquesInputs, true);
```
Comment (SamGondelman) [inline libraries/render-utils/src/RenderForwardTask.cpp]: You can move lines 99-103 down below too.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17516/HighFidelity-Beta-PR14988-aa8563f.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17519/HighFidelity-Beta-PR14988-109eb62.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17519/HighFidelity-Beta-PR14988-109eb62.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17519/HighFidelity-Beta-Interface-PR14988-109eb62.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17519/HighFidelity-Beta-PR14988-109eb62.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17519/HighFidelity-Beta-Interface-PR14988-109eb62.exe

**Pull request merged**